Lib/test/test_json/test_tool.py

```diff
@@ -100,9 +100,22 @@ def test_help_flag(self):
         self.assertEqual(err, b'')
 
     def test_sort_keys_flag(self):
-        infile = self._create_infile()
+        infile = self._create_infile(self.data)
         rc, out, err = assert_python_ok('-m', 'json.tool', '--sort-keys', infile)
         self.assertEqual(rc, 0)
         self.assertEqual(out.splitlines(),
                          self.expect_without_sort_keys.encode().splitlines())
         self.assertEqual(err, b'')
+
+    def test_no_ascii_flag(self):
```

Comment:
Done in https://github.com/python/cpython/pull/201/commits/a3e2b23b1115e66444469685b57e534f0cef69ff.